buildenv/jenkins/docker-nodes/jenkinsfile-build-container.groovy

```diff
@@ -42,9 +42,9 @@ if ("${OS}" == "unknown" || !(validArch.contains(ARCH))) {
 }
 
 def BUILD_OPTS = ""
-if ("${OS}" == "centos6.9") {
+if ("${OS}" == "centos6.10") {
     if ("${ARCH}" == "x86") {
-        BUILD_OPTS = "--dist=centos --version=6.9"
+        BUILD_OPTS = "--dist=centos --version=6.10 --cuda"
```

Comment:
CUDA should be selected for ppc64le and x86, independent of the O/S.

Reply:
I'll work in for all platforms.

Reply:
I decided against enabling cuda for all platforms here as the files extracted would be from ub16.  I doubt many will be wanting to use ub16 as a container as its out of support.  UB18+ and RH7+ we should likely aim for a version of cuda that would more closely align with the arch and os release.  I think that should be done under a different PR.

Reply:
It's only copying some header files from that image; only the version of CUDA should be relevant, but it should probably be updated to a supported OS version.

Reply:
I expect we want to stick with version 9.0 for a while, which is only available on ubuntu 16.04; see https://hub.docker.com/r/nvidia/cuda/tags?page=1&ordering=last_updated&name=9.0.